caas/kubernetes/provider/cloud.go

```diff
@@ -306,93 +306,36 @@ func (p kubernetesEnvironProvider) FinalizeCloud(ctx environs.FinalizeCloudConte
 	return cld, nil
 }
 
-func checkMicrok8sUserGroupSetup(cmdRunner CommandRunner) error {
```

Comment:
For running juju by developers directly from a locally compiled binary, we should keep this probably